src/components/Input/Input.tsx

```diff
@@ -1,12 +1,26 @@
 import React, { forwardRef } from 'react';
-import PropTypes from 'prop-types';
 import bem from '../../utils/bem';
 import styles from './Input.scss';
-import { CONTEXTS, INPUT_TYPES, SIZES } from '../../constants';
+import { ContextTypes, InputTypes, SizesTypes } from '../../constants';
+
+// Any other attributes (onChange, onKeyUp etc.) are
+// supported although not defined in props type definition
+interface Props extends Omit<React.InputHTMLAttributes<HTMLInputElement>, 'size'> {
+    /** The input field context (e.g. brand, primary, bad, good etc. - defaults to brand) */
+    context: ContextTypes;
```

Comment:
in Storybook these now show up as `type: unknown`. Is it something we could remedy?

Reply:
I found it a quite important regression and this should be brought up to the whole team and we need to all agree that on if/how to go ahead with it.